lib/lwm2m_carrier/include/lwm2m_os.h

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef LWM2M_OS_H__
+
+#include <stddef.h>
+#include <stdint.h>
+
+/*
+ * @brief Maximum number of timers that system must support.
```

Comment:
This should probably be "Minimum number of timers that the system must support."?

Reply:
I think that this is actually correct, that's the maximum number of times that the OS must support in order to run this library.